tools/crmadmin.c

```diff
@@ -251,7 +268,8 @@ do_work(void)
         cib_t *the_cib = cib_new();
         xmlNode *output = NULL;
 
-        int rc = the_cib->cmds->signon(the_cib, crm_system_name, cib_command);
+        int rc = the_cib->cmds->signon(the_cib,
+            ipc_name?ipc_name:crm_system_name, cib_command);
```

Comment:
It would simplify a bit to do "if (ipc_name == NULL) ipc_name = crm_system_name" after parsing args